apps/emqx/src/emqx_cm.erl

```diff
@@ -620,11 +620,19 @@ lookup_client({username, Username}) ->
     ],
     ets:select(emqx_channel_info, MatchSpec);
 lookup_client({clientid, ClientId}) ->
-    [
-        Rec
-     || Key <- ets:lookup(emqx_channel, ClientId),
-        Rec <- ets:lookup(emqx_channel_info, Key)
-    ].
+    case ets:lookup(emqx_channel, ClientId) of
```

Comment:
Maybe should fix the issue that client id doesn't exist in `emqx_channel` rather than looking up another table?